openpype/hosts/tvpaint/lib.py

```diff
@@ -677,6 +710,8 @@ def rename_filepaths_by_frame_start(
     # Skip if source first frame is same as destination first frame
     new_dst_filepaths = {}
     for src_frame, dst_frame in zip(source_range, output_range):
+        if not filepaths_by_frame.get(src_frame, False):
```

Comment:
if not filepaths_by_frame.get(src_frame)

False est inutile

Reply:
yes